torchrec/csrc/dynamic_embedding/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+add_library(tde_cpp_objs
+            OBJECT
+            details/clz_impl.cpp
+            details/ctz_impl.cpp)
+
+target_include_directories(tde_cpp_objs PUBLIC ${CMAKE_SOURCE_DIR})
```

Comment:
Do not use CMAKE_SOURCE_DIR. Because it makes the CMake project cannot be included as a sub-project.

Reply:
fixed.